src/site/content/en/patterns/example-set/index.md

```diff
@@ -0,0 +1,7 @@
+---
+layout: pattern-set
+title: Example PatternSet
+description: Example set of patterns
+hero: image/tcFciHGuF3MxnTr1y5ue01OGLBn2/dYAbN2LdttJ1qYk1gssh.svg
+draft: true
```

Comment:
Could you instead create "example-set.11tydata.js" in this folder which sets `draft: true` for everything? Or would you prefer it set on invididual files in case they're copied and pasted for new demos?

Reply:
We need this to be in frontmatter and not in 11tydata file, because it is used by parsing directly the frontmatter from the file, before any of 11ty magic kicks in (in /lib/patterns.js).